test/battle/hold_effect/light_ball.c

```diff
@@ -1,4 +1,99 @@
 #include "global.h"
 #include "test/battle.h"
 
-TO_DO_BATTLE_TEST("TODO: Write Light Ball (Hold Effect) test titles")
+ASSUMPTIONS
+{
+    ASSUME(GetItemHoldEffect(ITEM_LIGHT_BALL) == HOLD_EFFECT_LIGHT_BALL);
+}
+
+static const u16 speciesToCheck[] = {
```

Comment:
Any reason this is an ``u16``?

Reply:
> Any reason this is an `u16`?

Thought it would use less space, but I suppose it doesn't matter much. I'll change it to u32